front/src/components/boxs/chart/style.css

```
@@ -80,6 +80,22 @@
   text-decoration: none;
 }
 
+/* The loader icon is a static glyph: it only looks like a loader when it turns. */
+.spinning {
+  display: inline-block;
+  animation: dropdownItemChartSpin 1s linear infinite;
+}
+
+@keyframes dropdownItemChartSpin {
```

Stylelint (17.14.0)

[error] Expected keyframe name "dropdownItemChartSpin" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)